roles/servicetelemetry/tasks/main.yml

```diff
@@ -28,24 +28,33 @@
   - not is_openshift
   - not is_k8s
 
+- name: Print some debug information
+  vars:
+    msg: |
+      ServiceTelemetry Variables
+      --------------------------------------------
+      {{ servicetelemetry_vars | to_nice_yaml }}
+  debug:
+    msg: "{{ msg.split('\n') }}"
+
 - name: Adjust defaults for HA
-  when: high_availability_enabled
+  when: high_availability.enabled
   set_fact:
-    elasticsearch_node_count: 3
-    elasticsearch_redundancy_policy: FullRedundancy
-    qdr_deployment_size: 2
+    servicetelemetry_vars.events.elasticsearch.node_count: 3
```

Comment:
This is a lot more verbose, darn near java-like, but I think it's worth it.

Reply:
We could always change `servicetelemetry_vars` to `stv` or something if we hated the verbosity :)